src/components/workspace/controlBar/linkButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { GOOGLE_CLIENT_ID } from '../../../utils/constants';
+
+import { IconNames } from '@blueprintjs/icons';
+
+import { controlButton } from '../../commons';
+
+export type linkButtonProps = {
+  key: string;
+};
+
+export function LinkButton(props: linkButtonProps) {
```

Comment:
this function is a little unreadable. Could we maybe split this up and store the url as a constant and another function to format the actual url please!

Reply:
fixed

Reply:
I have stored the url as constants! It should be ok now.